### PR TITLE
Fix log uploading for the virtual-kubelet backend

### PR DESCRIPTION
**Pull request by gabrielhartmann**
The virtual kubelet entrypoint wasn't setting up log uploading.

### CODE REVIEW

File: cmd/virtual-kubelet-backend-titus-executor/main.go [hunk 1]
```diff
@@ -89,9 +90,16 @@ func mainWithError(podFileName string, statusPipe string, dockerCfg *docker.Conf
 	defer pipe.Close()
 	log.G(ctx).WithField("pod", pod.Name).Debugf("Got pipe %v", statusPipe)
 
-	logUploaders := uploader.NewUploadersFromUploaderArray([]uploader.Uploader{})
+	m := metrics.Discard
```
Comment (sargun) [inline cmd/virtual-kubelet-backend-titus-executor/main.go]: Can you put a TODO here?

File: cmd/virtual-kubelet-backend-titus-executor/main.go [hunk 2]
```diff
@@ -89,9 +90,16 @@ func mainWithError(podFileName string, statusPipe string, dockerCfg *docker.Conf
...
 
-	dockerRunner, err := runner.New(ctx, metrics.Discard, logUploaders, *cfg, *dockerCfg)
+	log.G(ctx).WithField("pod", pod.Name).Debugf("Getting uploaders from %+v", cfg.S3Uploaders)
+	var logUploaders *uploader.Uploaders
+	if logUploaders, err = uploader.NewUploaders(cfg, m); err != nil {
+		return fmt.Errorf("cannot create log uploaders: %v", err)
```
Comment (sargun) [inline cmd/virtual-kubelet-backend-titus-executor/main.go]: use errors.wrapf

Comment (sargun) [reply cmd/virtual-kubelet-backend-titus-executor/main.go]: err, errors.wrap

File: cmd/virtual-kubelet-backend-titus-executor/main.go (hunk 1 above)
Comment (gabrielhartmann) [reply cmd/virtual-kubelet-backend-titus-executor/main.go]: I filed a JIRA.

File: cmd/virtual-kubelet-backend-titus-executor/main.go (hunk 2 above)
Comment (gabrielhartmann) [reply cmd/virtual-kubelet-backend-titus-executor/main.go]: [You first](https://github.com/Netflix/titus-executor/blob/master/cmd/titus-executor/main.go#L119-L142).

**Pull request merged**